server/plugin.go

```diff
@@ -88,6 +94,11 @@ func (p *Plugin) OnActivate() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to initialize own command handling")
 	}
+
+	if err := p.api.Admin.SynchronizeApps(); err != nil {
```

Comment:
0/5 s/SynchronizeApps/LoadAppsList/?  Synchronize does not appear to be specifically intention-revealing.